fastify.js

```diff
@@ -404,6 +405,15 @@ function build (options) {
     fourOhFour.router.lookup(req, res)
   }
 
+  function onBadUrl (path, req, res) {
+    const body = `{"error":"Bad Request","message":"'${path}' is not a valid url component","statusCode":400}`
+    res.writeHead(400, {
+      'Content-Type': 'application/json',
```

Comment:
Shouldn't this be configurable? Some people may not be using Fastify as an `application/json` only server.

Reply:
Good point, but we are already doing the same in other parts of the framework.
https://github.com/fastify/fastify/blob/0aef7f323a79835ae6529a4f72c90744a731c74e/lib/route.js#L291-L297

https://github.com/fastify/fastify/blob/0aef7f323a79835ae6529a4f72c90744a731c74e/fastify.js#L389-L395

This can be a point of discussion, would you mind opening an issue?

Reply:
See https://github.com/fastify/fastify/issues/1930